recipes/prov/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "1.4.0" %}
+
+package:
+  name: prov
+  version: {{ version }}
+
+source:
+  fn: prov-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/prov/prov-{{ version }}.tar.gz
+  md5: 251576e384ab87eb68bee84697e863a3
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - python-dateutil
+    - networkx >=1.10
+    - lxml
+    - pydotplus
+    - six >=1.9.0
+
+  run:
+    - python
+    - python-dateutil
+    - networkx >=1.10
+    - lxml
+    - pydotplus
+    - six >=1.9.0
+
+test:
+  # Python imports
+  imports:
+    - prov
+    - prov.serializers
+    - prov.tests
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  requires:
+    - pydotplus
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: https://github.com/trungdong/prov
+  license: MIT License
```

Comment:
In general, we normally drop the word "License" from the `license` metadata as it is redundant.